backends/midi/alsa.cpp

```diff
@@ -285,6 +286,12 @@ void MidiDriver_ALSA::sysEx(const byte *msg, uint16 length) {
 	memcpy(buf + 1, msg, length);
 	buf[length + 1] = 0xF7;
 
+	// FIXME: Wait 100ms to see if it helps (it does when using amidi and sysex; anything slower checksum errors)
+	// FIXME: Slows down but still has checksum error. is bad data real? Didn't slow down enough
+	// FIXME: Per llamamusic's D-110 page: 390ms interval required.  Guess I'm making this configurable as many other synth nerds will not be pleased with that much delay
+	// FIXME: High system load can result in missing the sysex patch load before moving on to the title screen in KQ4; will need to find a way to alert the engine to pause
+	usleep(39 * 1000); // 39ms
```

Comment:
We have our own cross-platform functionality for `sleep()`. Check `g_system->delayMillis()`